From c18e7f9230e4975a811e660393af721e5092d73f Mon Sep 17 00:00:00 2001
Message-Id: <c18e7f9230e4975a811e660393af721e5092d73f@dist-git>
From: Andrea Bolognani <abologna@redhat.com>
Date: Wed, 5 Aug 2015 18:18:19 +0200
Subject: [PATCH] tests: Restore links in deconfigured-cpus nodeinfo test

When cleaning up the data (taken from a running system) for inclusion
I went a little too far and deleted a bunch of links that should have
been left alone. The test worked despite this because it was going
through a fallback code path.

A few other files are affected as well: again, the data is taken from
a running system, so even thought we would probably be okay if we
just added the links, aligning everything is definitely safer.

(cherry picked from commit cc3d52eb2f42a7220557f21ac124cf8dea5d607c)

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1213713

Signed-off-by: Andrea Bolognani <abologna@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 .../cpu/cpu128/topology/core_id                    |  2 +-
 .../cpu/cpu16/topology/core_id                     |  2 +-
 .../cpu/cpu24/topology/core_id                     |  2 +-
 .../cpu/cpu48/topology/core_id                     |  2 +-
 .../cpu/cpu72/topology/core_id                     |  2 +-
 .../linux-deconfigured-cpus/node/node0/cpu0        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu1        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu10       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu100      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu101      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu102      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu103      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu11       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu12       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu13       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu14       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu144      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu145      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu146      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu147      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu148      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu149      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu15       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu150      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu151      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu152      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu153      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu154      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu155      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu156      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu157      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu158      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu159      |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu16       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu17       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu18       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu19       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu2        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu20       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu21       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu22       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu23       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu24       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu25       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu26       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu27       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu28       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu29       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu3        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu30       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu31       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu32       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu33       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu34       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu35       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu36       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu37       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu38       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu39       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu4        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu5        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu56       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu57       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu58       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu59       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu6        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu60       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu61       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu62       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu63       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu64       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu65       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu66       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu67       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu68       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu69       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu7        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu70       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu71       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu8        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu80       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu81       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu82       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu83       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu84       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu85       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu86       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu87       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu88       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu89       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu9        |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu90       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu91       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu92       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu93       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu94       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu95       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu96       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu97       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu98       |  1 +
 .../linux-deconfigured-cpus/node/node0/cpu99       |  1 +
 .../linux-deconfigured-cpus/node/node0/meminfo     | 36 ++++++++++----------
 .../linux-deconfigured-cpus/node/node1/cpu40       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu41       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu42       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu43       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu44       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu45       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu46       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu47       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu48       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu49       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu50       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu51       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu52       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu53       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu54       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu55       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu72       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu73       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu74       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu75       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu76       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu77       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu78       |  1 +
 .../linux-deconfigured-cpus/node/node1/cpu79       |  1 +
 .../linux-deconfigured-cpus/node/node1/meminfo     | 36 ++++++++++----------
 .../linux-deconfigured-cpus/node/node16/cpu104     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu105     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu106     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu107     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu108     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu109     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu110     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu111     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu112     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu113     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu114     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu115     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu116     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu117     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu118     |  1 +
 .../linux-deconfigured-cpus/node/node16/cpu119     |  1 +
 .../linux-deconfigured-cpus/node/node16/meminfo    | 36 ++++++++++----------
 .../linux-deconfigured-cpus/node/node17/cpu120     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu121     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu122     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu123     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu124     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu125     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu126     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu127     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu128     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu129     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu130     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu131     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu132     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu133     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu134     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu135     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu136     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu137     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu138     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu139     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu140     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu141     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu142     |  1 +
 .../linux-deconfigured-cpus/node/node17/cpu143     |  1 +
 .../linux-deconfigured-cpus/node/node17/meminfo    | 38 +++++++++++-----------
 169 files changed, 238 insertions(+), 78 deletions(-)
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu0
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu1
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu10
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu100
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu101
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu102
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu103
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu11
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu12
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu13
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu14
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu144
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu145
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu146
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu147
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu148
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu149
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu15
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu150
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu151
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu152
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu153
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu154
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu155
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu156
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu157
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu158
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu159
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu16
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu17
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu18
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu19
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu2
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu20
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu21
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu22
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu23
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu24
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu25
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu26
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu27
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu28
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu29
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu3
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu30
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu31
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu32
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu33
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu34
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu35
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu36
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu37
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu38
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu39
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu4
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu5
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu56
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu57
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu58
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu59
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu6
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu60
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu61
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu62
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu63
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu64
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu65
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu66
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu67
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu68
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu69
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu7
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu70
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu71
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu8
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu80
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu81
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu82
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu83
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu84
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu85
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu86
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu87
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu88
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu89
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu9
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu90
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu91
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu92
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu93
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu94
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu95
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu96
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu97
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu98
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu99
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu40
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu41
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu42
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu43
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu44
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu45
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu46
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu47
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu48
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu49
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu50
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu51
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu52
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu53
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu54
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu55
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu72
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu73
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu74
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu75
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu76
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu77
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu78
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu79
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu104
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu105
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu106
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu107
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu108
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu109
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu110
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu111
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu112
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu113
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu114
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu115
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu116
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu117
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu118
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu119
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu120
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu121
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu122
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu123
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu124
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu125
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu126
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu127
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu128
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu129
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu130
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu131
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu132
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu133
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu134
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu135
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu136
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu137
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu138
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu139
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu140
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu141
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu142
 create mode 120000 tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu143

diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu128/topology/core_id b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu128/topology/core_id
index 6edc5b6..5c54d10 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu128/topology/core_id
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu128/topology/core_id
@@ -1 +1 @@
-2224
+2216
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu16/topology/core_id b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu16/topology/core_id
index f906e18..21e72e8 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu16/topology/core_id
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu16/topology/core_id
@@ -1 +1 @@
-96
+48
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu24/topology/core_id b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu24/topology/core_id
index b16e5f7..f906e18 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu24/topology/core_id
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu24/topology/core_id
@@ -1 +1 @@
-104
+96
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu48/topology/core_id b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu48/topology/core_id
index 1057e9a..de8febe 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu48/topology/core_id
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu48/topology/core_id
@@ -1 +1 @@
-176
+168
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu72/topology/core_id b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu72/topology/core_id
index eb08bc0..7c022ae 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu72/topology/core_id
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/cpu/cpu72/topology/core_id
@@ -1 +1 @@
-240
+232
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu0 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu0
new file mode 120000
index 0000000..c841bea
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu0
@@ -0,0 +1 @@
+../../cpu/cpu0
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu1 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu1
new file mode 120000
index 0000000..5f45362
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu1
@@ -0,0 +1 @@
+../../cpu/cpu1
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu10 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu10
new file mode 120000
index 0000000..a6dc6bb
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu10
@@ -0,0 +1 @@
+../../cpu/cpu10
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu100 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu100
new file mode 120000
index 0000000..ffb1446
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu100
@@ -0,0 +1 @@
+../../cpu/cpu100
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu101 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu101
new file mode 120000
index 0000000..2b399e6
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu101
@@ -0,0 +1 @@
+../../cpu/cpu101
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu102 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu102
new file mode 120000
index 0000000..f06eb7d
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu102
@@ -0,0 +1 @@
+../../cpu/cpu102
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu103 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu103
new file mode 120000
index 0000000..722b506
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu103
@@ -0,0 +1 @@
+../../cpu/cpu103
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu11 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu11
new file mode 120000
index 0000000..e29d898
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu11
@@ -0,0 +1 @@
+../../cpu/cpu11
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu12 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu12
new file mode 120000
index 0000000..211519e
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu12
@@ -0,0 +1 @@
+../../cpu/cpu12
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu13 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu13
new file mode 120000
index 0000000..52d3b69
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu13
@@ -0,0 +1 @@
+../../cpu/cpu13
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu14 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu14
new file mode 120000
index 0000000..24cb3b4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu14
@@ -0,0 +1 @@
+../../cpu/cpu14
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu144 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu144
new file mode 120000
index 0000000..ccecb10
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu144
@@ -0,0 +1 @@
+../../cpu/cpu144
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu145 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu145
new file mode 120000
index 0000000..00def86
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu145
@@ -0,0 +1 @@
+../../cpu/cpu145
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu146 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu146
new file mode 120000
index 0000000..6a13966
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu146
@@ -0,0 +1 @@
+../../cpu/cpu146
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu147 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu147
new file mode 120000
index 0000000..143c5a7
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu147
@@ -0,0 +1 @@
+../../cpu/cpu147
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu148 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu148
new file mode 120000
index 0000000..21bae94
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu148
@@ -0,0 +1 @@
+../../cpu/cpu148
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu149 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu149
new file mode 120000
index 0000000..fdb0f17
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu149
@@ -0,0 +1 @@
+../../cpu/cpu149
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu15 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu15
new file mode 120000
index 0000000..e905d85
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu15
@@ -0,0 +1 @@
+../../cpu/cpu15
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu150 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu150
new file mode 120000
index 0000000..da363e5
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu150
@@ -0,0 +1 @@
+../../cpu/cpu150
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu151 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu151
new file mode 120000
index 0000000..069de6b
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu151
@@ -0,0 +1 @@
+../../cpu/cpu151
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu152 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu152
new file mode 120000
index 0000000..0fb022e
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu152
@@ -0,0 +1 @@
+../../cpu/cpu152
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu153 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu153
new file mode 120000
index 0000000..d7e0f9e
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu153
@@ -0,0 +1 @@
+../../cpu/cpu153
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu154 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu154
new file mode 120000
index 0000000..0a6117a
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu154
@@ -0,0 +1 @@
+../../cpu/cpu154
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu155 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu155
new file mode 120000
index 0000000..201f0f5
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu155
@@ -0,0 +1 @@
+../../cpu/cpu155
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu156 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu156
new file mode 120000
index 0000000..da7f12b
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu156
@@ -0,0 +1 @@
+../../cpu/cpu156
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu157 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu157
new file mode 120000
index 0000000..c2aa5c2
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu157
@@ -0,0 +1 @@
+../../cpu/cpu157
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu158 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu158
new file mode 120000
index 0000000..4376692
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu158
@@ -0,0 +1 @@
+../../cpu/cpu158
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu159 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu159
new file mode 120000
index 0000000..852a0c4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu159
@@ -0,0 +1 @@
+../../cpu/cpu159
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu16 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu16
new file mode 120000
index 0000000..445de40
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu16
@@ -0,0 +1 @@
+../../cpu/cpu16
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu17 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu17
new file mode 120000
index 0000000..13929db
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu17
@@ -0,0 +1 @@
+../../cpu/cpu17
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu18 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu18
new file mode 120000
index 0000000..3809adc
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu18
@@ -0,0 +1 @@
+../../cpu/cpu18
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu19 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu19
new file mode 120000
index 0000000..c671083
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu19
@@ -0,0 +1 @@
+../../cpu/cpu19
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu2 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu2
new file mode 120000
index 0000000..2dcca33
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu2
@@ -0,0 +1 @@
+../../cpu/cpu2
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu20 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu20
new file mode 120000
index 0000000..a37c370
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu20
@@ -0,0 +1 @@
+../../cpu/cpu20
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu21 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu21
new file mode 120000
index 0000000..68a7cad
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu21
@@ -0,0 +1 @@
+../../cpu/cpu21
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu22 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu22
new file mode 120000
index 0000000..2c226ae
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu22
@@ -0,0 +1 @@
+../../cpu/cpu22
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu23 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu23
new file mode 120000
index 0000000..75c2f57
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu23
@@ -0,0 +1 @@
+../../cpu/cpu23
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu24 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu24
new file mode 120000
index 0000000..ad0a618
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu24
@@ -0,0 +1 @@
+../../cpu/cpu24
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu25 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu25
new file mode 120000
index 0000000..3145742
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu25
@@ -0,0 +1 @@
+../../cpu/cpu25
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu26 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu26
new file mode 120000
index 0000000..1dd180f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu26
@@ -0,0 +1 @@
+../../cpu/cpu26
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu27 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu27
new file mode 120000
index 0000000..74ced19
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu27
@@ -0,0 +1 @@
+../../cpu/cpu27
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu28 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu28
new file mode 120000
index 0000000..83269be
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu28
@@ -0,0 +1 @@
+../../cpu/cpu28
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu29 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu29
new file mode 120000
index 0000000..fa37dc6
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu29
@@ -0,0 +1 @@
+../../cpu/cpu29
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu3 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu3
new file mode 120000
index 0000000..c7690e5
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu3
@@ -0,0 +1 @@
+../../cpu/cpu3
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu30 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu30
new file mode 120000
index 0000000..80226fa
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu30
@@ -0,0 +1 @@
+../../cpu/cpu30
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu31 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu31
new file mode 120000
index 0000000..6a9a569
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu31
@@ -0,0 +1 @@
+../../cpu/cpu31
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu32 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu32
new file mode 120000
index 0000000..bbca866
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu32
@@ -0,0 +1 @@
+../../cpu/cpu32
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu33 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu33
new file mode 120000
index 0000000..b7a9f8d
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu33
@@ -0,0 +1 @@
+../../cpu/cpu33
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu34 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu34
new file mode 120000
index 0000000..0c0e738
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu34
@@ -0,0 +1 @@
+../../cpu/cpu34
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu35 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu35
new file mode 120000
index 0000000..47654d6
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu35
@@ -0,0 +1 @@
+../../cpu/cpu35
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu36 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu36
new file mode 120000
index 0000000..5e53f70
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu36
@@ -0,0 +1 @@
+../../cpu/cpu36
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu37 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu37
new file mode 120000
index 0000000..41b4ce7
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu37
@@ -0,0 +1 @@
+../../cpu/cpu37
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu38 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu38
new file mode 120000
index 0000000..7231eb4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu38
@@ -0,0 +1 @@
+../../cpu/cpu38
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu39 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu39
new file mode 120000
index 0000000..d465480
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu39
@@ -0,0 +1 @@
+../../cpu/cpu39
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu4 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu4
new file mode 120000
index 0000000..9e77a64
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu4
@@ -0,0 +1 @@
+../../cpu/cpu4
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu5 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu5
new file mode 120000
index 0000000..cc07c3b
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu5
@@ -0,0 +1 @@
+../../cpu/cpu5
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu56 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu56
new file mode 120000
index 0000000..586050f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu56
@@ -0,0 +1 @@
+../../cpu/cpu56
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu57 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu57
new file mode 120000
index 0000000..4276ac5
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu57
@@ -0,0 +1 @@
+../../cpu/cpu57
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu58 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu58
new file mode 120000
index 0000000..3c10d2d
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu58
@@ -0,0 +1 @@
+../../cpu/cpu58
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu59 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu59
new file mode 120000
index 0000000..06ddb97
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu59
@@ -0,0 +1 @@
+../../cpu/cpu59
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu6 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu6
new file mode 120000
index 0000000..2e75763
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu6
@@ -0,0 +1 @@
+../../cpu/cpu6
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu60 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu60
new file mode 120000
index 0000000..c062761
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu60
@@ -0,0 +1 @@
+../../cpu/cpu60
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu61 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu61
new file mode 120000
index 0000000..5a2c12f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu61
@@ -0,0 +1 @@
+../../cpu/cpu61
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu62 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu62
new file mode 120000
index 0000000..9056072
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu62
@@ -0,0 +1 @@
+../../cpu/cpu62
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu63 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu63
new file mode 120000
index 0000000..1511953
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu63
@@ -0,0 +1 @@
+../../cpu/cpu63
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu64 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu64
new file mode 120000
index 0000000..1664a2b
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu64
@@ -0,0 +1 @@
+../../cpu/cpu64
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu65 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu65
new file mode 120000
index 0000000..52712cf
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu65
@@ -0,0 +1 @@
+../../cpu/cpu65
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu66 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu66
new file mode 120000
index 0000000..93a4f32
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu66
@@ -0,0 +1 @@
+../../cpu/cpu66
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu67 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu67
new file mode 120000
index 0000000..db73e74
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu67
@@ -0,0 +1 @@
+../../cpu/cpu67
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu68 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu68
new file mode 120000
index 0000000..4f40eb8
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu68
@@ -0,0 +1 @@
+../../cpu/cpu68
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu69 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu69
new file mode 120000
index 0000000..9f32d96
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu69
@@ -0,0 +1 @@
+../../cpu/cpu69
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu7 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu7
new file mode 120000
index 0000000..09e3f79
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu7
@@ -0,0 +1 @@
+../../cpu/cpu7
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu70 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu70
new file mode 120000
index 0000000..a0e8fd2
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu70
@@ -0,0 +1 @@
+../../cpu/cpu70
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu71 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu71
new file mode 120000
index 0000000..898c9f2
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu71
@@ -0,0 +1 @@
+../../cpu/cpu71
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu8 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu8
new file mode 120000
index 0000000..bda10cc
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu8
@@ -0,0 +1 @@
+../../cpu/cpu8
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu80 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu80
new file mode 120000
index 0000000..ce5ed81
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu80
@@ -0,0 +1 @@
+../../cpu/cpu80
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu81 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu81
new file mode 120000
index 0000000..dcd9a05
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu81
@@ -0,0 +1 @@
+../../cpu/cpu81
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu82 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu82
new file mode 120000
index 0000000..c91ce01
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu82
@@ -0,0 +1 @@
+../../cpu/cpu82
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu83 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu83
new file mode 120000
index 0000000..22ba3c4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu83
@@ -0,0 +1 @@
+../../cpu/cpu83
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu84 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu84
new file mode 120000
index 0000000..c7e4d26
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu84
@@ -0,0 +1 @@
+../../cpu/cpu84
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu85 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu85
new file mode 120000
index 0000000..6779d27
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu85
@@ -0,0 +1 @@
+../../cpu/cpu85
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu86 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu86
new file mode 120000
index 0000000..9e777be
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu86
@@ -0,0 +1 @@
+../../cpu/cpu86
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu87 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu87
new file mode 120000
index 0000000..0359c58
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu87
@@ -0,0 +1 @@
+../../cpu/cpu87
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu88 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu88
new file mode 120000
index 0000000..3825c50
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu88
@@ -0,0 +1 @@
+../../cpu/cpu88
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu89 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu89
new file mode 120000
index 0000000..9a72211
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu89
@@ -0,0 +1 @@
+../../cpu/cpu89
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu9 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu9
new file mode 120000
index 0000000..1ec1db2
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu9
@@ -0,0 +1 @@
+../../cpu/cpu9
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu90 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu90
new file mode 120000
index 0000000..75d10b3
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu90
@@ -0,0 +1 @@
+../../cpu/cpu90
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu91 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu91
new file mode 120000
index 0000000..8e9e4a4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu91
@@ -0,0 +1 @@
+../../cpu/cpu91
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu92 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu92
new file mode 120000
index 0000000..f8d5f05
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu92
@@ -0,0 +1 @@
+../../cpu/cpu92
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu93 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu93
new file mode 120000
index 0000000..5a302ad
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu93
@@ -0,0 +1 @@
+../../cpu/cpu93
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu94 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu94
new file mode 120000
index 0000000..9e6239f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu94
@@ -0,0 +1 @@
+../../cpu/cpu94
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu95 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu95
new file mode 120000
index 0000000..d87b5e3
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu95
@@ -0,0 +1 @@
+../../cpu/cpu95
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu96 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu96
new file mode 120000
index 0000000..50d1edc
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu96
@@ -0,0 +1 @@
+../../cpu/cpu96
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu97 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu97
new file mode 120000
index 0000000..38b5a90
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu97
@@ -0,0 +1 @@
+../../cpu/cpu97
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu98 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu98
new file mode 120000
index 0000000..df97c6c
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu98
@@ -0,0 +1 @@
+../../cpu/cpu98
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu99 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu99
new file mode 120000
index 0000000..a3a6164
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/cpu99
@@ -0,0 +1 @@
+../../cpu/cpu99
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/meminfo b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/meminfo
index 1033a39..0ecf45a 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/meminfo
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node0/meminfo
@@ -1,28 +1,28 @@
 Node 0 MemTotal:       67108864 kB
-Node 0 MemFree:        61726720 kB
-Node 0 MemUsed:         5382144 kB
-Node 0 Active:           276544 kB
-Node 0 Inactive:         174848 kB
-Node 0 Active(anon):     100928 kB
-Node 0 Inactive(anon):     9152 kB
-Node 0 Active(file):     175616 kB
-Node 0 Inactive(file):   165696 kB
+Node 0 MemFree:        61960640 kB
+Node 0 MemUsed:         5148224 kB
+Node 0 Active:           124864 kB
+Node 0 Inactive:         107584 kB
+Node 0 Active(anon):      83648 kB
+Node 0 Inactive(anon):     8768 kB
+Node 0 Active(file):      41216 kB
+Node 0 Inactive(file):    98816 kB
 Node 0 Unevictable:           0 kB
 Node 0 Mlocked:               0 kB
-Node 0 Dirty:             69824 kB
+Node 0 Dirty:               128 kB
 Node 0 Writeback:             0 kB
-Node 0 FilePages:        360256 kB
-Node 0 Mapped:            42752 kB
-Node 0 AnonPages:         93312 kB
-Node 0 Shmem:             16768 kB
-Node 0 KernelStack:       10208 kB
-Node 0 PageTables:         1472 kB
+Node 0 FilePages:        155008 kB
+Node 0 Mapped:            31552 kB
+Node 0 AnonPages:         77312 kB
+Node 0 Shmem:             15104 kB
+Node 0 KernelStack:        8992 kB
+Node 0 PageTables:         1728 kB
 Node 0 NFS_Unstable:          0 kB
 Node 0 Bounce:                0 kB
 Node 0 WritebackTmp:          0 kB
-Node 0 Slab:             184576 kB
-Node 0 SReclaimable:      57152 kB
-Node 0 SUnreclaim:       127424 kB
+Node 0 Slab:             151872 kB
+Node 0 SReclaimable:      27776 kB
+Node 0 SUnreclaim:       124096 kB
 Node 0 AnonHugePages:         0 kB
 Node 0 HugePages_Total:     0
 Node 0 HugePages_Free:      0
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu40 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu40
new file mode 120000
index 0000000..c727d46
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu40
@@ -0,0 +1 @@
+../../cpu/cpu40
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu41 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu41
new file mode 120000
index 0000000..cc0b78c
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu41
@@ -0,0 +1 @@
+../../cpu/cpu41
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu42 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu42
new file mode 120000
index 0000000..8ebd18e
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu42
@@ -0,0 +1 @@
+../../cpu/cpu42
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu43 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu43
new file mode 120000
index 0000000..03b2d2d
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu43
@@ -0,0 +1 @@
+../../cpu/cpu43
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu44 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu44
new file mode 120000
index 0000000..859e4fc
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu44
@@ -0,0 +1 @@
+../../cpu/cpu44
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu45 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu45
new file mode 120000
index 0000000..58c3ca3
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu45
@@ -0,0 +1 @@
+../../cpu/cpu45
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu46 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu46
new file mode 120000
index 0000000..55223e0
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu46
@@ -0,0 +1 @@
+../../cpu/cpu46
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu47 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu47
new file mode 120000
index 0000000..19407d4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu47
@@ -0,0 +1 @@
+../../cpu/cpu47
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu48 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu48
new file mode 120000
index 0000000..60a190c
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu48
@@ -0,0 +1 @@
+../../cpu/cpu48
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu49 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu49
new file mode 120000
index 0000000..0ff1796
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu49
@@ -0,0 +1 @@
+../../cpu/cpu49
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu50 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu50
new file mode 120000
index 0000000..ddc5361
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu50
@@ -0,0 +1 @@
+../../cpu/cpu50
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu51 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu51
new file mode 120000
index 0000000..41163a9
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu51
@@ -0,0 +1 @@
+../../cpu/cpu51
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu52 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu52
new file mode 120000
index 0000000..be09dc8
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu52
@@ -0,0 +1 @@
+../../cpu/cpu52
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu53 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu53
new file mode 120000
index 0000000..470ab22
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu53
@@ -0,0 +1 @@
+../../cpu/cpu53
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu54 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu54
new file mode 120000
index 0000000..5c11dfd
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu54
@@ -0,0 +1 @@
+../../cpu/cpu54
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu55 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu55
new file mode 120000
index 0000000..e858949
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu55
@@ -0,0 +1 @@
+../../cpu/cpu55
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu72 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu72
new file mode 120000
index 0000000..d731cf5
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu72
@@ -0,0 +1 @@
+../../cpu/cpu72
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu73 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu73
new file mode 120000
index 0000000..99f1a90
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu73
@@ -0,0 +1 @@
+../../cpu/cpu73
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu74 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu74
new file mode 120000
index 0000000..84a3139
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu74
@@ -0,0 +1 @@
+../../cpu/cpu74
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu75 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu75
new file mode 120000
index 0000000..0693a99
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu75
@@ -0,0 +1 @@
+../../cpu/cpu75
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu76 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu76
new file mode 120000
index 0000000..5ff7021
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu76
@@ -0,0 +1 @@
+../../cpu/cpu76
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu77 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu77
new file mode 120000
index 0000000..05099c6
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu77
@@ -0,0 +1 @@
+../../cpu/cpu77
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu78 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu78
new file mode 120000
index 0000000..20eb9cc
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu78
@@ -0,0 +1 @@
+../../cpu/cpu78
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu79 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu79
new file mode 120000
index 0000000..4813a63
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/cpu79
@@ -0,0 +1 @@
+../../cpu/cpu79
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/meminfo b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/meminfo
index 600841b..ef085b2 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/meminfo
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node1/meminfo
@@ -1,28 +1,28 @@
 Node 1 MemTotal:       67108864 kB
-Node 1 MemFree:        66225792 kB
-Node 1 MemUsed:          883072 kB
-Node 1 Active:           247040 kB
-Node 1 Inactive:          82752 kB
-Node 1 Active(anon):      52800 kB
-Node 1 Inactive(anon):      192 kB
-Node 1 Active(file):     194240 kB
-Node 1 Inactive(file):    82560 kB
+Node 1 MemFree:        66143552 kB
+Node 1 MemUsed:          965312 kB
+Node 1 Active:           125952 kB
+Node 1 Inactive:         255808 kB
+Node 1 Active(anon):      79808 kB
+Node 1 Inactive(anon):      448 kB
+Node 1 Active(file):      46144 kB
+Node 1 Inactive(file):   255360 kB
 Node 1 Unevictable:           0 kB
 Node 1 Mlocked:               0 kB
-Node 1 Dirty:                64 kB
+Node 1 Dirty:            213760 kB
 Node 1 Writeback:             0 kB
-Node 1 FilePages:        281344 kB
-Node 1 Mapped:            19264 kB
-Node 1 AnonPages:         49088 kB
-Node 1 Shmem:              4352 kB
-Node 1 KernelStack:        2960 kB
-Node 1 PageTables:          448 kB
+Node 1 FilePages:        310656 kB
+Node 1 Mapped:            27328 kB
+Node 1 AnonPages:         73856 kB
+Node 1 Shmem:              6336 kB
+Node 1 KernelStack:        4624 kB
+Node 1 PageTables:          512 kB
 Node 1 NFS_Unstable:          0 kB
 Node 1 Bounce:                0 kB
 Node 1 WritebackTmp:          0 kB
-Node 1 Slab:             106368 kB
-Node 1 SReclaimable:      16448 kB
-Node 1 SUnreclaim:        89920 kB
+Node 1 Slab:             130688 kB
+Node 1 SReclaimable:      21120 kB
+Node 1 SUnreclaim:       109568 kB
 Node 1 AnonHugePages:         0 kB
 Node 1 HugePages_Total:     0
 Node 1 HugePages_Free:      0
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu104 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu104
new file mode 120000
index 0000000..7692606
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu104
@@ -0,0 +1 @@
+../../cpu/cpu104
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu105 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu105
new file mode 120000
index 0000000..77d25d5
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu105
@@ -0,0 +1 @@
+../../cpu/cpu105
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu106 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu106
new file mode 120000
index 0000000..f8bff26
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu106
@@ -0,0 +1 @@
+../../cpu/cpu106
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu107 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu107
new file mode 120000
index 0000000..d475c15
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu107
@@ -0,0 +1 @@
+../../cpu/cpu107
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu108 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu108
new file mode 120000
index 0000000..4352ab8
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu108
@@ -0,0 +1 @@
+../../cpu/cpu108
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu109 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu109
new file mode 120000
index 0000000..c74e8ec
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu109
@@ -0,0 +1 @@
+../../cpu/cpu109
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu110 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu110
new file mode 120000
index 0000000..fb66aa1
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu110
@@ -0,0 +1 @@
+../../cpu/cpu110
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu111 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu111
new file mode 120000
index 0000000..b2da00c
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu111
@@ -0,0 +1 @@
+../../cpu/cpu111
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu112 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu112
new file mode 120000
index 0000000..e65062a
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu112
@@ -0,0 +1 @@
+../../cpu/cpu112
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu113 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu113
new file mode 120000
index 0000000..f8380bf
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu113
@@ -0,0 +1 @@
+../../cpu/cpu113
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu114 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu114
new file mode 120000
index 0000000..4c33e35
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu114
@@ -0,0 +1 @@
+../../cpu/cpu114
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu115 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu115
new file mode 120000
index 0000000..2b4fbce
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu115
@@ -0,0 +1 @@
+../../cpu/cpu115
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu116 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu116
new file mode 120000
index 0000000..7e07a2d
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu116
@@ -0,0 +1 @@
+../../cpu/cpu116
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu117 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu117
new file mode 120000
index 0000000..2d65930
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu117
@@ -0,0 +1 @@
+../../cpu/cpu117
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu118 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu118
new file mode 120000
index 0000000..b09093c
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu118
@@ -0,0 +1 @@
+../../cpu/cpu118
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu119 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu119
new file mode 120000
index 0000000..59a5bf1
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/cpu119
@@ -0,0 +1 @@
+../../cpu/cpu119
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/meminfo b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/meminfo
index 78350e1..09e3954 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/meminfo
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node16/meminfo
@@ -1,28 +1,28 @@
 Node 16 MemTotal:       67108864 kB
-Node 16 MemFree:        66296832 kB
-Node 16 MemUsed:          812032 kB
-Node 16 Active:           145792 kB
-Node 16 Inactive:         131904 kB
-Node 16 Active(anon):      36352 kB
-Node 16 Inactive(anon):      320 kB
-Node 16 Active(file):     109440 kB
-Node 16 Inactive(file):   131584 kB
+Node 16 MemFree:        66573952 kB
+Node 16 MemUsed:          534912 kB
+Node 16 Active:            14272 kB
+Node 16 Inactive:          16832 kB
+Node 16 Active(anon):       6272 kB
+Node 16 Inactive(anon):       64 kB
+Node 16 Active(file):       8000 kB
+Node 16 Inactive(file):    16768 kB
 Node 16 Unevictable:           0 kB
 Node 16 Mlocked:               0 kB
-Node 16 Dirty:             48256 kB
+Node 16 Dirty:                 0 kB
 Node 16 Writeback:             0 kB
-Node 16 FilePages:        246720 kB
-Node 16 Mapped:             7744 kB
-Node 16 AnonPages:         35968 kB
-Node 16 Shmem:               704 kB
-Node 16 KernelStack:        3232 kB
-Node 16 PageTables:          512 kB
+Node 16 FilePages:         26176 kB
+Node 16 Mapped:              384 kB
+Node 16 AnonPages:          4928 kB
+Node 16 Shmem:              1408 kB
+Node 16 KernelStack:        1968 kB
+Node 16 PageTables:          192 kB
 Node 16 NFS_Unstable:          0 kB
 Node 16 Bounce:                0 kB
 Node 16 WritebackTmp:          0 kB
-Node 16 Slab:              61760 kB
-Node 16 SReclaimable:      28736 kB
-Node 16 SUnreclaim:        33024 kB
+Node 16 Slab:              46336 kB
+Node 16 SReclaimable:       2688 kB
+Node 16 SUnreclaim:        43648 kB
 Node 16 AnonHugePages:         0 kB
 Node 16 HugePages_Total:     0
 Node 16 HugePages_Free:      0
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu120 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu120
new file mode 120000
index 0000000..17a18b1
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu120
@@ -0,0 +1 @@
+../../cpu/cpu120
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu121 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu121
new file mode 120000
index 0000000..321bbc2
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu121
@@ -0,0 +1 @@
+../../cpu/cpu121
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu122 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu122
new file mode 120000
index 0000000..e03f5a3
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu122
@@ -0,0 +1 @@
+../../cpu/cpu122
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu123 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu123
new file mode 120000
index 0000000..3021e4f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu123
@@ -0,0 +1 @@
+../../cpu/cpu123
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu124 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu124
new file mode 120000
index 0000000..10b32dc
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu124
@@ -0,0 +1 @@
+../../cpu/cpu124
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu125 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu125
new file mode 120000
index 0000000..9e06e6e
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu125
@@ -0,0 +1 @@
+../../cpu/cpu125
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu126 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu126
new file mode 120000
index 0000000..2a4708a
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu126
@@ -0,0 +1 @@
+../../cpu/cpu126
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu127 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu127
new file mode 120000
index 0000000..96b5143
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu127
@@ -0,0 +1 @@
+../../cpu/cpu127
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu128 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu128
new file mode 120000
index 0000000..3142217
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu128
@@ -0,0 +1 @@
+../../cpu/cpu128
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu129 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu129
new file mode 120000
index 0000000..dc84401
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu129
@@ -0,0 +1 @@
+../../cpu/cpu129
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu130 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu130
new file mode 120000
index 0000000..59a50b8
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu130
@@ -0,0 +1 @@
+../../cpu/cpu130
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu131 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu131
new file mode 120000
index 0000000..ca1f958
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu131
@@ -0,0 +1 @@
+../../cpu/cpu131
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu132 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu132
new file mode 120000
index 0000000..a1f7ea1
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu132
@@ -0,0 +1 @@
+../../cpu/cpu132
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu133 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu133
new file mode 120000
index 0000000..89e453b
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu133
@@ -0,0 +1 @@
+../../cpu/cpu133
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu134 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu134
new file mode 120000
index 0000000..a46c33e
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu134
@@ -0,0 +1 @@
+../../cpu/cpu134
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu135 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu135
new file mode 120000
index 0000000..4efd76f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu135
@@ -0,0 +1 @@
+../../cpu/cpu135
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu136 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu136
new file mode 120000
index 0000000..cfd61b4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu136
@@ -0,0 +1 @@
+../../cpu/cpu136
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu137 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu137
new file mode 120000
index 0000000..6a1ec92
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu137
@@ -0,0 +1 @@
+../../cpu/cpu137
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu138 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu138
new file mode 120000
index 0000000..890aad3
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu138
@@ -0,0 +1 @@
+../../cpu/cpu138
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu139 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu139
new file mode 120000
index 0000000..afb925f
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu139
@@ -0,0 +1 @@
+../../cpu/cpu139
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu140 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu140
new file mode 120000
index 0000000..2faf4c0
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu140
@@ -0,0 +1 @@
+../../cpu/cpu140
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu141 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu141
new file mode 120000
index 0000000..6cf8cf1
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu141
@@ -0,0 +1 @@
+../../cpu/cpu141
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu142 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu142
new file mode 120000
index 0000000..fa73d61
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu142
@@ -0,0 +1 @@
+../../cpu/cpu142
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu143 b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu143
new file mode 120000
index 0000000..04dcaa4
--- /dev/null
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/cpu143
@@ -0,0 +1 @@
+../../cpu/cpu143
\ No newline at end of file
diff --git a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/meminfo b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/meminfo
index 3e691f2..1bddfb5 100644
--- a/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/meminfo
+++ b/tests/nodeinfodata/linux-deconfigured-cpus/node/node17/meminfo
@@ -1,29 +1,29 @@
 Node 17 MemTotal:       67108864 kB
-Node 17 MemFree:        65890688 kB
-Node 17 MemUsed:         1218176 kB
-Node 17 Active:           250752 kB
-Node 17 Inactive:         181184 kB
-Node 17 Active(anon):     100544 kB
-Node 17 Inactive(anon):      320 kB
-Node 17 Active(file):     150208 kB
-Node 17 Inactive(file):   180864 kB
+Node 17 MemFree:        66334656 kB
+Node 17 MemUsed:          774208 kB
+Node 17 Active:            15936 kB
+Node 17 Inactive:          12416 kB
+Node 17 Active(anon):      10944 kB
+Node 17 Inactive(anon):       64 kB
+Node 17 Active(file):       4992 kB
+Node 17 Inactive(file):    12352 kB
 Node 17 Unevictable:           0 kB
 Node 17 Mlocked:               0 kB
-Node 17 Dirty:             97856 kB
+Node 17 Dirty:                 0 kB
 Node 17 Writeback:             0 kB
-Node 17 FilePages:        341824 kB
-Node 17 Mapped:             2304 kB
-Node 17 AnonPages:        100736 kB
-Node 17 Shmem:               576 kB
-Node 17 KernelStack:        3088 kB
-Node 17 PageTables:          768 kB
+Node 17 FilePages:         17600 kB
+Node 17 Mapped:              768 kB
+Node 17 AnonPages:         10752 kB
+Node 17 Shmem:               256 kB
+Node 17 KernelStack:        3808 kB
+Node 17 PageTables:          128 kB
 Node 17 NFS_Unstable:          0 kB
 Node 17 Bounce:                0 kB
 Node 17 WritebackTmp:          0 kB
-Node 17 Slab:              61760 kB
-Node 17 SReclaimable:      14912 kB
-Node 17 SUnreclaim:        46848 kB
-Node 17 AnonHugePages:     49152 kB
+Node 17 Slab:              25792 kB
+Node 17 SReclaimable:       3840 kB
+Node 17 SUnreclaim:        21952 kB
+Node 17 AnonHugePages:         0 kB
 Node 17 HugePages_Total:     0
 Node 17 HugePages_Free:      0
 Node 17 HugePages_Surp:      0
-- 
2.5.0

